sci-electronics/fritzing/fritzing-0.9.4.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg-utils
+
+MY_PV="CD-498"
+DESCRIPTION="Electronic Design Automation"
+HOMEPAGE="https://fritzing.org/"
+SRC_URI="https://github.com/fritzing/fritzing-app/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="CC-BY-SA-3.0 GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="dev-qt/qtconcurrent:5=
+	dev-qt/qtcore:5=
+	dev-qt/qtgui:5=
+	dev-qt/qtnetwork:5=
+	dev-qt/qtprintsupport:5=
+	dev-qt/qtserialport:5=
+	dev-qt/qtsql:5=[sqlite]
+	dev-qt/qtsvg:5=
+	dev-qt/qtwidgets:5=
+	dev-qt/qtxml:5=
+	>=dev-libs/quazip-0.7.2[qt5(+)]
+	dev-libs/libgit2:=
+"
+DEPEND="
+	${RDEPEND}
+	>=dev-libs/boost-1.40
+"
+
+S="${WORKDIR}/${PN}-app-${MY_PV}"
+
+DOCS="README.md"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-libgit2-version.patch"
+	"${FILESDIR}/${P}-user-repos-dir.patch"
+)
+
+src_prepare() {
+	local lang translations=
+
+	# fix build with newer quazip - bug #597988
+	sed -i -e "s/#include <quazip/&5/" src/utils/folderutils.cpp || die
+	sed -i -e "s|/usr/include/quazip|&5|" -e "s/-lquazip/&5/" phoenix.pro || die
+
+	# Get a rid of the bundled libs
+	# Bug 412555 and
+	# https://code.google.com/p/fritzing/issues/detail?id=1898
+	rm -rf src/lib/quazip/ pri/quazip.pri src/lib/boost* || die
+
+	# Fritzing doesn't need zlib
+	sed -i -e 's:LIBS += -lz::' -e 's:-lminizip::' phoenix.pro || die
+
+	# Use system libgit
+	sed -i -e 's:LIBGIT_STATIC.*:LIBGIT_STATIC = false:' phoenix.pro || die
+
+	# Somewhat evil but IMHO the best solution
+	for lang in $L10N; do
+		lang=${lang/linguas_}
+		[[ -f "translations/${PN}_${lang}.qm" ]] && translations+=" translations/${PN}_${lang}.qm"
+	done
+	if [[ -n "${translations}" ]]; then
+		sed -i -e "s:\(translations.extra =\) .*:\1	cp -p ${translations} \$(INSTALL_ROOT)\$\$PKGDATADIR/translations\r:" phoenix.pro || die
+	else
+		sed -i -e "s:translations.extra = .*:\r:" phoenix.pro || die
+	fi
+
+	default
+}
+
+src_configure() {
+	eqmake5 DEFINES=QUAZIP_INSTALLED phoenix.pro
+}
+
+src_install() {
+	INSTALL_ROOT="${D}" default
+}
+
+pkg_postinst() {
+	einfo
+	einfo "Fritzing requires parts repository to function properly. To download, run"
+	einfo "\"git clone -b master https://github.com/fritzing/fritzing-parts.git\""
```

Comment:
I'm a little bit lost here. Is there a reason we can't include this in the ebuild?

Reply:
Technically we can, but the new fritzing feature introduced in this version uses libgit to update its parts repo directly from upstream, which requires the parts directory to be a git repo and modifiable by users. So it's a bit tricky indeed.